### PR TITLE
Change Singleton package Name to Microsoft.WinAppRuntime.Singleton[-shorttag]

### PR DESCRIPTION
**Pull request by DrusTheAxe**
Store package Name required for Store package signing

Also, add missing DDLM constants

https://task.ms/39401667

### CODE REVIEW

File: specs/Deployment/MSIXPackageVersioning.md [hunk 1]
```diff
@@ -403,6 +403,7 @@ and release 1.x...
 
 * WARfwk: `Microsoft.WindowsAppRuntime.<rmajor>.<rminor>[-tag]`
 * WARmain: `MicrosoftCorporationII.WinAppRuntime.Main.<rmajor>.<rminor>[-shorttag]`
+* WARmain: `MicrosoftCorporationII.WinAppRuntime.Singleton[-shorttag]`
```
Comment (sachintaMSFT) [inline specs/Deployment/MSIXPackageVersioning.md:406]: WARsingleton (not WARmain).

btw, should WAR be changed to MSIX or WAS, here too ?

File: specs/Deployment/MSIXPackageVersioning.md [hunk 2]
```diff
@@ -515,7 +516,7 @@ length constraints. The specific packages Names in Windows App SDK 1.0:
 
 * WARfwk: `Microsoft.WindowsAppRuntime.<rmajor>.<rminor>[-tag]`
 * WARmain: `MicrosoftCorporationII.WinAppRuntime.Main.<rmajor>.<rminor>[-shorttag]`
-* WARmain: `Microsoft.WindowsAppRuntime.Singleton[-tag]`
+* WARmain: `MicrosoftCorporationII.WinAppRuntime.Singleton[-shorttag]`
 * WARddlm: `Microsoft.WinAppRuntime.DDLM.<major>.<minor>.<build>.<revision>-<shortarchitecture>[-shorttag]`
```
Comment (sachintaMSFT) [inline specs/Deployment/MSIXPackageVersioning.md:520]: Same. WARsingleton

**Review by BenJKuhn — APPROVED** [review]

**Review by sachintaMSFT — APPROVED** [review]

File: specs/Deployment/MSIXPackageVersioning.md (hunk 1 above)
Comment (sachintaMSFT) [reply specs/Deployment/MSIXPackageVersioning.md]: Fixed naming WARmain to WARsingleton for all singleton package names in the file

**Comment by kythant** [comment]
You completed this PR right before I forked the release/1.1-stable branch FYI
No need to CP
https://github.com/microsoft/WindowsAppSDK/commits/release/1.1-stable

**Pull request merged**